Automattic/vip-go-mu-plugins
# Cleans up some phpcs issues

## Description

Just a small cleanup I was in mood for.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check the CI build
